src/components/common/MockDatabase.js

```diff
@@ -0,0 +1,68 @@
+export const MOCK_DATABASE = [
+  {
+    title: "Airmax",
+    images: [
```

Comment:
do you need to pull. this shouldn't be a diff here

Reply:
Sorry, I didn't understand. I already did pull and I couldn't see any `diff` in `Files changed` tab regarding this file neither. 
![image](https://user-images.githubusercontent.com/30778803/89068168-5e034e80-d379-11ea-8339-8e0b1b9bf18f.png)